packages/sw-runtime-caching/src/lib/stale-while-revalidate.js

```diff
@@ -24,7 +24,10 @@ import assert from '../../../../lib/assert';
  *
  * In addition to updating the appropriate caches, it will also trigger any
  * appropriate plugins defined in the underlying `RequestWrapper`.
- * XXXXXXXX
+ * 
+ * This class always returns the cache and returns it first regardless of how
+ * quickly a network response returns. This make it suitable for scenarios where
+ * a fastest strategy would otherwise be used.
```

Comment:
@jeffposnick does the current behavior of staleWhileRevalidate() match the old description from https://googlechrome.github.io/sw-toolbox/api.html#toolboxfastest? If so, I wonder if there's value in linking back to the legacy implementation of this inline in comments for those who are used to using toolbox.

```
Request the resource from both the cache and the network in parallel. Respond with whichever returns first. Usually this will be the cached version, if there is one. On the one hand this strategy will always make a network request, even if the resource is cached. On the other hand, if/when the network request completes the cache is updated, so that future cache reads will be more up-to-date.
```